doc/source/citation.md

```diff
@@ -0,0 +1,114 @@
+(citation)=
+
+# Citing and method overview
+
+When using a method implemented in xDEM, one should cite both the package and the original study behind the method (if there is any)!
+
+## Citing xDEM
+
+To cite the package, use the Zenodo DOI: [![Zenodo](https://zenodo.org/badge/doi/10.5281/zenodo.4809697.svg)](https://zenodo.org/doi/10.5281/zenodo.4809697).
+
+## Method overview
+
+For citation and other purposes, here's an overview of all methods implemented in the package and their reference, if it exists.
+More details are available on each feature page!
+
+### Terrain attributes
+
+```{list-table}
+   :widths: 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Method
+     - Reference
+   * - Slope, aspect and hillshade
+     - [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) or [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107)
+   * - Curvatures
+     - [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107)
+   * - Topographic position index
+     - [Weiss (2001)](http://www.jennessent.com/downloads/TPI-poster-TNC_18x22.pdf)
+   * - Terrain ruggedness index
+     - [Riley et al. (1999)](http://download.osgeo.org/qgis/doc/reference-docs/Terrain_Ruggedness_Index.pdf) or [Wilson et al. (2007)](http://dx.doi.org/10.1080/01490410701295962)
+   * - Roughness
+     - [Dartnell (2000)](https://environment.sfsu.edu/node/11292)
+   * - Rugosity
+     - [Jenness (2004)](<https://doi.org/10.2193/0091-7648(2004)032[0829:CLSAFD]2.0.CO;2>)
+   * - Fractal roughness
+     - [Taud and Parrot (2005)](https://doi.org/10.4000/geomorphologie.622)
+```
+
+### Coregistration
+
+```{list-table}
+   :widths: 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Method
+     - Reference
+   * - Nuth and Kääb
+     - [Nuth and Kääb (2011)](https://doi.org/10.5194/tc-5-271-2011)
+   * - Dh minimization
+     - N/A
+   * - Iterative closest point
+     - [Besl and McKay (1992)](https://doi.org/10.1117/12.57955)
+   * - Vertical shift
+     - N/A
+```
+
+### Bias-correction
+
+```{list-table}
+   :widths: 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Method
+     - Reference
+   * - Deramp
+     - N/A
+   * - Directional bias (sum of sinuoids)
+     - [Girod et al. (2017)](https://doi.org/10.3390/rs9070704)
+   * - Directional bias (other)
+     - N/A
+   * - Terrain bias (maximum curvature)
```

Comment:
Hopefully we can update this with the work of Arnaud!

Reply:
Yes! :slightly_smiling_face:

Reply:
I thought exactly the same when redacting it.